requirements_all.txt

```diff
@@ -1272,7 +1272,7 @@ pyspcwebgw==0.4.0
 pystride==0.1.7
 
 # homeassistant.components.sensor.syncthru
-pysyncthru==0.3.1
+pysyncthru==0.4.1
```

Comment:
You forgot to run the script to have this file updated.

Reply:
right, fixed